### PR TITLE
Wehelp Stage2 Week4 Tasks

### PR DESCRIPTION
**Pull request by 54Laichu**
賴益楚
http://54.157.160.232:8000
http://54.157.160.232:8000/attraction/2

### CODE REVIEW

**Review by cwpeng — CHANGES_REQUESTED** [review]
Reload page after successful signing in.

File: static/js/userAuth.js [hunk 1]
```diff
@@ -68,7 +68,9 @@ document.addEventListener('DOMContentLoaded', function () {
         if (response.status === 200 && result.token) {
           localStorage.setItem('token', result.token);
           showMessage('登入成功', 'green');
-          signInButton.innerHTML = '<p>登出系統</p>';
```
Comment (54Laichu) [inline static/js/userAuth.js]: if user login successfully, reload current page

File: static/js/userAuth.js [hunk 2]
```diff
@@ -68,7 +68,9 @@ document.addEventListener('DOMContentLoaded', function () {
...
+          setTimeout(() => {
+            window.location.reload();
```
Comment (54Laichu) [inline static/js/userAuth.js:72]: if user login successfully, reload current page

File: static/js/userAuth.js [hunk 3]
```diff
@@ -68,7 +68,9 @@ document.addEventListener('DOMContentLoaded', function () {
...
+          }, 1000);
```
Comment (54Laichu) [inline static/js/userAuth.js:73]: if user login successfully, reload current page

**Review by cwpeng — APPROVED** [review]
OK. Done.

**Pull request merged**